content/en/docs/overview/singleton-sdk/Angular Client Introduction/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Angular Client Introduction"
+date: 2019-09-24T20:08:31+08:00
+draft: false
+weight: 30
+---
+
+singleton for Angular client is common lib developed by VMware G11n team to provide i18n support for Angular framework. The client will communicate with singleton server for fetching translation and i18n patterns.
+
+Most translations provided by singleton service comes from Human translators, when new application is connected to singleton service, source strings (English strings), required language and related information will be sent to singleton service, singleton service doesn’t return real translation immediately since the human translators need time to figure out the quilted translation, at this moment, singleton service just delivers pseudo tag as placeholder to indicate singleton client works well with singleton service and App itself in development stage. Once the translations are ready, when the client side calls singleton service again, the real translations will be delivered via HTTP request. Please note that under English locale, the client doesn't make a network call to singleton service, but return English directly.
+
+Besides Localization, singleton client also provides I18n support for Datetime format, number format, currency and plural. singleton I18n service is based on CLDR data repository and keep the same scope in supported locales set. For the specific usages and API details please refer to following sections, if you find below APIs can't cover all usages of your application requirement, please feel free to contact with our team to add related API.
+
+
+
+#### **Features**
+
+- Level-2 Dates, Times, Numbers and Currencies formatting support. For more details refer to  [i18n Level-2](https://confluence.eng.vmware.com/display/GQ/I18n+Level-2).
+- Level-3 support. For more details refer to [i18n Level-3](https://confluence.eng.vmware.com/display/GQ/I18n+Level-3).
+
+
+#### **Version**
+
+- 8.x (support Angular8)
+- 7.x (support Angular7.x)
+- 6.x (support Angular6.x)
+- 1.x (support Angular4.x & Angular5.x)
+- 0.7.x is no longer maintained, only for bugfix
+
+>"VIPServiceConfig" has been removed from Version 8.x and the singleton host information are listed here: [Singleton instances](https://ngx.eng.vmware.com/@vmw/ngx-vip/vip-instance/documentation). For more changes in Version 8.x, please check the changelog.
```

Comment:
It should redirect to singleton-instances.md

Reply:
I have fixed it now, it redirects to singleton-instances.md.